.circleci/config.yml

```diff
@@ -20,8 +20,8 @@ jobs:
           key: dependency-cache-{{ checksum "package.json" }}
       - run: npm install
       - run: npm run build
-      - run: npm run test:cover
-      - run: bash <(curl -s https://codecov.io/bash)
+      - run: npm run test:cover-unit
+      - run: bash <(curl -s https://codecov.io/bash) -F unittests
```

Comment:
Not really a new change, but I'm always a bit concerned seeing curl to bash scripts.  
Maybe we could eventually use the node reporter? Looks like we do that anyway in the `Build Success` step for the integration tests.